docs/configuration.md

```diff
@@ -374,6 +374,27 @@ The [Lighthouse settings object](https://github.com/GoogleChrome/lighthouse/blob
 }
 ```
 
+You can use the `settings` property to configure Lighthouse for testing your site on an emulated desktop device:
```

Comment:
WDYT of putting this down below in the [Common Examples section](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/configuration.md#common-examples) where we have a longer list of examples? 

I worry that including too many inline might detract from the primary goal of explaining what each LHCI property does.

Reply:
Oh right, I didn't see that section previously. I've moved it now!